content/examples/prometheus/pomerium-alerts.yml

```diff
@@ -0,0 +1,271 @@
+          service: autocert
+        annotations:
+          summary: "Autocert certificate expiring soon"
+          description: "Next certificate expires in {{ $value | humanizeDuration }}"
+          runbook: |
+            1. Check autocert renewal logs
+            2. Verify ACME provider connectivity
+            3. Check domain DNS configuration
+            4. Verify autocert storage accessibility
+
+      - alert: PomeriumAutocertRenewalFailures
+        expr: increase(pomerium_autocert_renewals_total[1h]) == 0 and pomerium_autocert_certificate_next_expires_seconds - time() < 2592000
+        for: 2h
+        labels:
+          severity: critical
+          component: pomerium
+          service: autocert
+        annotations:
+          summary: "Autocert certificate renewal failures"
+          description: "No certificate renewals in the last hour with certificates expiring soon"
+          runbook: |
+            1. Check autocert logs for renewal errors
+            2. Verify ACME provider status and rate limits
+            3. Check domain ownership and DNS configuration
+            4. Verify autocert storage permissions
+
+      - alert: PomeriumIdentityManagerSessionRefreshFailures
+        expr: rate(pomerium_identity_manager_session_refresh_errors[5m]) > 0.1
+        for: 5m
+        labels:
+          severity: warning
+          component: pomerium
+          service: identity-manager
+        annotations:
+          summary: "High session refresh failure rate"
+          description: "Session refresh failure rate is {{ $value }} failures/sec"
+          runbook: |
+            1. Check identity provider connectivity and health
+            2. Verify OAuth2/OIDC configuration
+            3. Check token refresh endpoint availability
+            4. Review identity provider rate limits
+
+      - alert: PomeriumIdentityManagerUserRefreshFailures
+        expr: rate(pomerium_identity_manager_last_user_refresh_errors[5m]) > 0.1
+        for: 5m
+        labels:
+          severity: warning
+          component: pomerium
+          service: identity-manager
+        annotations:
+          summary: "High user refresh failure rate"
+          description: "User refresh failure rate is {{ $value }} failures/sec"
+          runbook: |
+            1. Check identity provider user endpoint availability
+            2. Verify API permissions and scopes
+            3. Check identity provider rate limits
+            4. Review user synchronization configuration
+
+      - alert: PomeriumIdentityManagerStaleData
+        expr: time() - pomerium_identity_manager_last_refresh_timestamp > 3600
+        for: 10m
+        labels:
+          severity: warning
+          component: pomerium
+          service: identity-manager
+        annotations:
+          summary: "Identity manager data is stale"
+          description: "Identity data hasn't been refreshed in over 1 hour"
+          runbook: |
+            1. Check logs for errors
```

Comment:
Let's clarify that we are referring to the dataplane logs, and if there's a filter we can put on that, it's good to write that in.